ALevelHomework6.0/ALevelHomework6.0/Program.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+
+internal class Products
+{
+    public int Number { get; set; }
+    public string Name { get; set; }
+    public decimal Price { get; set; }
+}
+
+internal class ShopCart
+{
+    public Products[] Components { get; set; }
+
+    public ShopCart(int ability)
+    {
+        Components = new Products[ability];
+    }
+}
```

Comment:
I don't understand this class, we need at least properties of  id, sum, date so on